agent/acs/handler/acs_handler.go

```diff
@@ -185,7 +180,7 @@ func acsWsURL(endpoint, cluster, containerInstanceArn string, taskEngine engine.
 	query.Set("containerInstanceArn", containerInstanceArn)
 	query.Set("agentHash", version.GitHashString())
 	query.Set("agentVersion", version.Version)
-	query.Set("seqNum", strconv.FormatInt(SequenceNumber.Get(), 10))
+	query.Set("seqNum", strconv.FormatInt(1, 10))
```

Comment:
Can you add a unit test or extend the existing one for this? https://github.com/aws/amazon-ecs-agent/blob/dev/agent/acs/handler/acs_handler_test.go#L56